index.html

```diff
@@ -0,0 +1,521 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Vrushabh Dhatrak" />
+    <meta name="author" content="Vrushabh Dhatrak" />
```

Comment:
repeated the same line

Reply:
Sorry my bad that for description & content of the page I did that